samples/subsys/settings/default/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _settings-sample:
```

Comment:
These document labels must be unique across all documents (the sample documentation above has the same label).  You'll need to change one of them.  That said, the words in these documents are identical (except for the document title).  What's up?

Reply:
Hi @dbkinder, thanks for the review. I have changed the labels to settings-sample-default and settings-sample-custom. The difference between the two samples is that the default sample uses a default backend (selected by the config file), the second sample does a setup of a non-default backend that can be more suited to the needs.